### DISCUSSION TITLE
Support of development and donations

### DISCUSSION THREAD

**Discussion by mavlac** [doublecmd/doublecmd#802] [Suggestions]
I've been happily using Double Commander for a long and I would like to support the active development, but I am unable to find a way to do so. If there is one, would you consider adding it to the software itself, or the homepage, or even here to readme file?

**Comment by alexx2000** [comment]
No way to do it.

**Reply by ags1234** [reply]
What do you mean by this? Other open source projects are open to donations.

**Reply by alexx2000** [reply]
I mean that no way to make donation at this moment.

**Comment by mmtaco** [comment]
I’d like to make a donation to support the development of Double Commander. Could you please let me know the preferred donation method (PayPal, bank transfer, etc.) and how the funds will be used?
